### PR TITLE
Add suis pp_db; ggcaller to publications

### PR DESCRIPTION
**Pull request by johnlees**
Tidy and update other pages

### CODE REVIEW

**Comment by johnlees** [comment]
@samhorsfield96 can you check you are happy with the ggcaller description I've added – please edit as necessary! We can also add a proper blog post after it's published

File: content/group/index.md [hunk 1]
```diff
@@ -26,7 +26,24 @@ I am a PhD student at the Wellcome Sanger Institute in [Prof. Stephen Bentley’
...
 _raymond at ebi dot ac dot uk_
 
-**Joel Hellewell** is a postdoctoral fellow
+**Berk Gonenc** is a Master's student
+
+{{< figure src="/images/group/berk_gonenc.jpg" >}}
+
+My background includes wet-lab as well as bioinformatics. I have a BSc in Molecular Biology and Genetics from the respected Bogazici University and worked in a NGS analysis bioinformatics company ([Genomize](https://genomize.com)) for nearly 2 years. I am now doing my MSc in "Biosciences and Bioengineering Innovations for Precision Medicine ([BeInPM](https://www.beinprecisionmedicine.eu))" at the University Grenoble Alpes.
+
+My internship will focus on developing a bayesian model of bacterial population. I’m motivated to expand my knowledge on statistics and data analysis under Dr. John Lees supervision at EMBL-EBI.
```
Comment (samhorsfield96) [inline content/group/index.md:35]: *populations

Comment (samhorsfield96) [inline content/group/index.md:35]: *populations

File: content/publications/index.md [hunk 2]
```diff
@@ -12,6 +12,45 @@ A list of some of the papers directly from our group, with some short explanatio
...
+
+### Accurate and fast graph-based pangenome annotation and clustering with ggCaller
+
+Samuel T. Horsfield, Nicholas J. Croucher and John A. Lees (2023). **Accurate and fast graph-based pangenome annotation and clustering with ggCaller**
+*bioRxiv*
+
+https://www.biorxiv.org/content/10.1101/2023.01.24.524926v1
+
+Annotation of bacterial pangenomes is typically done one at a time. Each input assembly
+is annotated by calling and scoring ORFs, and aligning all of these against databases
```
Comment (samhorsfield96) [inline content/publications/index.md:25]: It may be a good idea just to highlight what ORF and COG stand for for the uninitiated.

**Review by samhorsfield96 — CHANGES_REQUESTED** [review]
Just a couple of comments, otherwise all good!

**Comment by samhorsfield96** [comment]
All good!

**Review by samhorsfield96 — APPROVED** [review]
All good!

**Pull request merged**